Cargo.toml

```diff
@@ -84,8 +84,6 @@ backend_libinput = ["input"]
 backend_session = []
 backend_udev = ["udev", "input/udev"]
 backend_vulkan = ["ash", "scopeguard"]
-backend_session_logind = ["dbus", "backend_session", "pkg-config"]
```

Comment:
The `dbus` optional dependency should be unneeded now.

Reply:
good call, removed!